pokemongo_bot/event_handlers/discord_handler.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+from pokemongo_bot.event_manager import EventHandler
+from pokemongo_bot.base_dir import _base_dir
+import json
+import os
+import time
+import discord_simple
+import thread
+import re
+from pokemongo_bot.datastore import Datastore
+import pprint
+
+
+DEBUG_ON = False
+
+class FileIOException(Exception):
+    pass
+
+class DiscordClass:
+
+    update_id = None
+
+
+    def __init__(self, bot, master, pokemons, config):
+        self.bot = bot
+        self.pokemons = pokemons
+        self._dbot = None
+        self.config = config
+
+    def sendMessage(self, to=None, parse_mode='Markdown', text=None):
+        self._dbot.send_message(to, text)
+
+    def connect(self):
+        self._dbot = discord_simple.Bot(self.bot.config.discord_token,on_message=self.on_message)
+
+    def _get_player_stats(self):
+        web_inventory = os.path.join(_base_dir, "web", "inventory-%s.json" % self.bot.config.username)
+        try:
+            with open(web_inventory, "r") as infile:
+                json_inventory = json.load(infile)
+        except ValueError as exception:
+            self.bot.logger.info('[x] Error while opening inventory file for read: %s' % exception)
+            json_inventory = []
+        except:
+            raise FileIOException("Unexpected error reading from {}".format(web_inventory))
+        return next((x["inventory_item_data"]["player_stats"]
+                     for x in json_inventory
+                     if x.get("inventory_item_data", {}).get("player_stats", {})),
+                    None)
+
```

Comment:
There is no need to reload inventory from json. Just access the appropriate Inventory (inventory.player_stats from memory)